LondonBritishLibrary/orient/BLorient8820.xml

```diff
@@ -0,0 +1,283 @@
+                                    <title ref="LIT2509Weddas#Sunday"/>
+                                </msItem>
+                            </msItem>
+                            <msItem xml:id="ms_i1.4">
+                                <locus from="182vb" to="189rb"/>
+                                <title ref="LIT1113Anqasa"/>
+                            </msItem>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">2+189</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>156</height>
+                                        <width>150</width>
+                                    </dimensions>
+                                </extent></supportDesc>
+                            <layoutDesc>
+                                <layout columns="1" writtenLines="17 18"><locus from="1r" to="171v"/></layout>
+                                <layout columns="2" writtenLines="17 18"><locus from="171v" to="189r"/></layout>
+                            </layoutDesc>
+                        </objectDesc> 
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <desc>Written in good hand.</desc>
+                                <date notBefore="1700" notAfter="1900">18th to 19th century</date>
+                            </handNote>
+                        </handDesc>
+                        <decoDesc>
+                            <decoNote xml:id="d1" type="band">
+                                <locus target="#1r #48r #98v"/>
+                                <desc>Ornamental bands in red and black ink.</desc>
+                            </decoNote>
+                        </decoDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <locus target="#IIv"/>
+                                    <desc type="StampExlibris">Written in pencil.</desc>
+                                    <q xml:lang="la">Psalterium aethiopicae</q>
+                                </item>
+                                <item xml:id="e2">
+                                    <locus target="#189r"/>
+                                    <desc type="Condemnation">The usual anathema against thieves and persons who might erase the manuscript, 
+                                        containing the name of the owner, <persName role="owner" ref="PRS14372Yohannes">Yoḥannǝs</persName>,
```

Comment:
I wonder if it may be worth creating a narrative unit for the curse against thieves @thea-m @DenisNosnitsin1970 ?

Reply:
I think it is a good idea!

Reply:
@CarstenHoffmannMarburg can you create a NAR record for the curse? thank you!